database/insertData.js

```diff
@@ -316,9 +316,23 @@ const queries = [
                 }
               }
               {
-                actionCode: "changeStatus"
+                actionCode: "grantPermissions"
                 trigger: ON_APPLICATION_SUBMIT
                 sequence: 3
+                parameterQueries: {
+                  username: {
+                    operator: "objectProperties"
+                    children: [{ value: { property: "responses.Q3.text" } }]
```

Comment:
Personally, I prefer to use the output of the previous Action to provide the username, but this works fine in this case, so not to worry.

Reply:
I don't think actions should be coupled and reliant on each other, even in configurations